src/main/java/seedu/dengue/model/overview/AgeOverview.java

```diff
@@ -0,0 +1,80 @@
+package seedu.dengue.model.overview;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.dengue.logic.analyst.AgeAnalyst;
+import seedu.dengue.logic.analyst.DataBin;
+import seedu.dengue.model.person.Person;
+
+/**
+ * An {@code Overview} that runs analysis by {@code Age}.
+ *
+ * @see AgeAnalyst
+ */
+public class AgeOverview extends Overview {
+    private static final String AGE_TITLE = "Overview by Age";
+    private static final int MAX_AGE_LENGTH = 9; // "321 - 123"
```

Comment:
not sure if comment should remain here

Reply:
Thanks for pointing it out! I'll remove them in the next PR.